src/wizard/android/index.md

```diff
@@ -116,9 +116,9 @@ transaction.finish() // Mark the transaction as finished and send it to Sentry
 
 Check out [the documentation](https://docs.sentry.io/platforms/android/performance/instrumentation/) to learn more about the API and automatic instrumentations.
 
-> Want to play with some new features? Try out our beta [Room performance integration](https://docs.sentry.io/platforms/android/performance/instrumentation/automatic-instrumentation/#sqlite-and-room-instrumentation).
+> Want to play with some new features? Try out our beta [Room](https://docs.sentry.io/platforms/android/configuration/integrations/room-and-sqlite/) and [File I/O](https://docs.sentry.io/platforms/android/configuration/integrations/file-io/) performance integrations.
 >  
-> This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/gradle); you must use version `3.0.0-beta.1`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures performance of the database queries done with Room if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
+> This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/gradle); you must use version `3.0.0-beta.3`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures the performance of the database queries done with Room as well as file I/O operations if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
```

Comment:
Should the "F" in "file I/O" be capitalized here?
```suggestion
> This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/gradle); you must use version `3.0.0-beta.3`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures the performance of the database queries done with Room as well as file I/O operations if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
```

Reply:
haha, that's exactly what @philipphofmann has changed, but I also think it should be capitalized here

Reply:
Ha! Ultimately, it just needs to be consistent. It's capitalized throughout the File I/O documentation and always capitalize all the other integration names, so I'm not sure why we wouldn't do that for this one.